packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2786,6 +2789,11 @@ export type ParticipantsRemovedListener = (event: {
 // @public
 export type ParticipantState = 'Idle' | 'Connecting' | 'Ringing' | 'Connected' | 'Hold' | 'InLobby' | 'EarlyMedia' | 'Disconnected';
 
+// @public
+export type Profile = {
```

Comment:
will this hold more in the future like email and such for a contact card?

Reply:
Yep, that is why we named this as Profile, and once major update is allowed, we should add AvatarPersonaData here as well